lnd_test.go

```diff
@@ -3659,8 +3659,12 @@ func testSpiderShortestPath(net *lntest.NetworkHarness, t *harnessTest) {
 		return channelInfo.Channels, nil
 	}*/
 	
+	// Set channel sizes here. chanAmt sets the channel
 	const chanAmt = btcutil.Amount(200000)
 	const pushAmt = btcutil.Amount(100000)
+
+	baseFee := int64(0)
+	feeRate := int64(1)
```

Comment:
can you add a comment explaining these two fields? In particular, what is feeRate?